Input/Content/Translation.md

```diff
@@ -6,7 +6,7 @@ The models were evaluated by ratio plots of area under the plasma concentration-
 
 
 
-Eq. 1: GMFE=10^((Σ|log10(pred PK parameter∕obs PK parameter)|)∕n)
+Eq. 1: $$ {\Huge  GMFE = 10^{\frac{\sum(|log(\frac{Pred}{Obs})|)}{n}} }  $$
```

Comment:
Replace `$$ `with `$`.
When having additional text in the same line (here: `Eq. 1:`), $ must be used
Already created the same issue for 1A9: https://github.com/Open-Systems-Pharmacology/Pediatric_Qualification_Package_UGT1A9_Ontogeny/issues/8